src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] Args) {
```

Comment:
The `Args` parameter should be named using lowerCamelCase, according to Java naming conventions. Consider renaming it to `args`.